syntax = "proto3";
import "google/protobuf/empty.proto";

// to compile
//python3 -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. --pyi_out=. ./gRPC/PROTO/load_balancer.proto


message AirAnalysisResponse {
    float temperature = 1;
    float humidity = 2;
    double time = 3;
}

message PollutionAnalysisResponse {
    float co2 = 1;
    double time = 2;
}


service LoadBalancerServicer {

  rpc ReceiveMeteo(AirAnalysisResponse) returns (google.protobuf.Empty) {}
  rpc ReceivePollution(PollutionAnalysisResponse) returns (google.protobuf.Empty) {}



}

